app/scripts/directives/util.js

```diff
@@ -149,6 +149,22 @@ angular.module('openshiftConsole')
       }
     };
   })
+  .directive('setHomePage', function($uibModal) {
+    return {
+      restrict: 'E',
+      replace: true,
+      template: '<a href="">Set Home Page</a>',
+      link: function($scope, element) {
+        element.bind('click', function() {
```

Comment:
Why not use `ng-click`?

Reply:
The directive is being used in an extension.  I did try ng-click in the nav-menu controller but the extensions seem to not trigger the ng-click function in the controller.

Reply:
OK I see the problem and it's what we do for copy login link. I'm OK with it.

Reply:
Is a bug with extensions?  I believe there is a way to `ng-click`, but it's been a good while since I've looked @ the code.  Doesn't need to hold up this PR, but I can def take a look @ it if its causing trouble.